SemanticFormsSelect.php

```diff
@@ -68,11 +68,11 @@ public static function initExtension() {
 	public static function onExtensionFunction() {
 
 		if ( !defined( 'SF_VERSION' ) ) {
-			die( '<b>Error:</b><a href="https://github.com/SemanticMediaWiki/SemanticFormsSelect/">Semantic Forms Select</a> requires the <a href="https://www.mediawiki.org/wiki/Extension:SemanticForms">Semantic Forms</a> extension. Please install and activate this extension first.' );
+			die( '<b>Error:</b><a href="https://github.com/SemanticMediaWiki/SemanticFormsSelect/">Semantic Forms Select</a> requires the <a href="https://www.mediawiki.org/wiki/Extension:PageForms">Semantic Forms</a> extension. Please install and activate this extension first.' );
```

Comment:
Hi yep missed that one, that was fixed by @mwjames in https://github.com/SemanticMediaWiki/SemanticFormsSelect/commit/483f3702a7daa9f3b98a6f9b7daa2288b6c8978e